src/openms/include/OpenMS/QC/TIC.h

```diff
@@ -71,7 +70,16 @@ namespace OpenMS
     @param bin_size RT bin size in seconds
     @return TIC Chromatogram
     **/
-    void compute(const MSExperiment &exp, float bin_size=0);
+    struct Result
```

Comment:
the struct is now annotated with the docs of `compute()`.
Intended? :)

Reply:
don't think so ;)